src/initialize.F90

```diff
@@ -1036,38 +1034,33 @@ recursive subroutine allocate_offsets(univ_list, counts, found)
     integer, intent(out), allocatable     :: counts(:,:)  ! Target count
     logical, intent(out), allocatable     :: found(:,:)   ! Target found
 
-    integer :: i, j, k, l, m                    ! Loop counters
-    type(SetInt)               :: cell_list     ! distribells to track
-    type(Universe),    pointer :: univ          ! pointer to universe
-    class(Lattice),    pointer :: lat           ! pointer to lattice
-    type(TallyObject), pointer :: t             ! pointer to tally
-    type(TallyFilter), pointer :: filter        ! pointer to filter
+    integer      :: i, j, k   ! Loop counters
+    type(SetInt) :: cell_list ! distribells to track
```

Comment:
You should add a call to `cell_list % clear()` at the end of this subroutine. Otherwise we have a (small) memory leak.